Integration package naming throws linting errors

If you run the linter, you'll see issues like:

```
tests/integration/integration_test.go:1:9: package should be `integration_test` instead of `integration` (testpackage)
package integration // nolint:typecheck
        ^
tests/integration/utils.go:1:21: directive `// nolint:typecheck` is unused for linter "typecheck" (nolintlint)
package integration // nolint:typecheck
                    ^
make: *** [Makefile:50: verify-go-lint] Error 1
```

This is because the current tests are in `tests/integration/` but the package name is `integration_test`.

--- a/tests/helpers.go
+++ b/tests/helpers.go
@@ -1,4 +1,4 @@
-package integration_test // nolint:typecheck
+package tests
 
 type Subject struct {
 	ID   string

--- a/tests/integration_test.go
+++ b/tests/integration_test.go
@@ -1,4 +1,4 @@
-package integration_test // nolint:typecheck
+package tests // nolint:testpackage
 
 import (
 	"database/sql"
@@ -72,7 +72,7 @@ func getMigrationHelper(t *testing.T) *migrate.Migrate {
 	if err != nil {
 		t.Skip("Unable to initialize database driver for migrations")
 	}
-	m, err := migrate.NewWithDatabaseInstance("file://../../migrations", "postgres", driver)
+	m, err := migrate.NewWithDatabaseInstance("file://../migrations", "postgres", driver)
 	if err != nil {
 		t.Skip("Unable to initialize migrations")
 	}

